src/switch_ivr.c

```diff
@@ -793,13 +793,35 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_parse_event(switch_core_session_t *se
 	return switch_channel_test_flag(channel, CF_BREAK) ? SWITCH_STATUS_BREAK : status;
 }
 
+static void log_event_data(switch_core_session_t *session, switch_event_t *event)
+{
+	char *cmd = switch_event_get_header(event, "call-command");
+	switch_ssize_t hlen = -1;
+	unsigned long cmd_hash;
+	unsigned long CMD_EXECUTE = switch_hashfunc_default("execute", &hlen);
+
+	if (zstr(cmd)) {
+		return;
+	}
+
+	cmd_hash = switch_hashfunc_default(cmd, &hlen);
```

Comment:
Since the log_event_data will only be called if its from the audio_bridge_function, then I think we can just log the whole event instead as it might be possible for it not a `call-command` or a `CMD_EXECUTE`